site/docs/pages/transaction/transaction.mdx

```diff
@@ -15,6 +15,8 @@ import { Wallet, ConnectWallet } from '@coinbase/onchainkit/wallet';
 import App from '../../components/App';
 import TransactionWrapper from '../../components/TransactionWrapper';
 
+export const PAYMASTER_AND_BUNDLER_ENDPOINT = import.meta.env.VITE_CDP_PAYMASTER_AND_BUNDLER_ENDPOINT;
```

Comment:
I had to export this variable otherwise MDX would cause it to fail with:

` Unexpected VariableDeclaration in code: only import/exports are supportedMDXmicromark-extension-mdxjs-esm:non-esm
`